packages/financial-templates-lib/test/clients/ExpiringMultiPartyClient.js

```diff
@@ -246,16 +213,16 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
 
           client = new ExpiringMultiPartyClient(
             dummyLogger,
-            financialContractInstance.abi,
+            financialContract.abi,
             web3,
             emp.address,
             testConfig.collateralDecimals,
             testConfig.syntheticDecimals,
             testConfig.priceFeedDecimals,
-            currentTypeTested // either ExpiringMultiParty OR Perpetual depending on the test
+            contractVersion.contractType // either ExpiringMultiParty OR Perpetual depending on the test
           );
         });
-        versionedIt(["any"])("Returns all positions", async function() {
+        versionedIt([{ contractType: "any", contractVersion: "any" }])("Returns all positions", async function() {
```

Comment:
versions are now defined as an object with `contractType` and `contractVersion` as keys, like defined in the common util.